drivers/hmis/app/graphql/schema.graphql

```diff
@@ -5652,6 +5653,63 @@ type KeyValue {
   value: String
 }
 
+type LastContact {
+  date: ISO8601Date!
+  type: LastContactType!
```

Comment:
`contactDate` and `contactType` would make this more descriptive and searchable. With such generic names "date" and "type" it can be tricky to find usages in the codebase. (Sorry, I should've suggested that on my first review)